README.md

```diff
@@ -56,6 +56,12 @@ You should take this generated password and populate it in the `DB_PASS` of your
 - `DB_NAME` should be set to `address_book`.
 - `SITE_URL` should be set to the address from which the system will be accessible from. Typically http://localhost/ is acceptable.
 
+5. (OPTIONAL) To define a timezone orther than the default 'Europe/London', in your `docker-compose.yml` add the following under the 'php-fpm' service:
```

Comment:
If it's included in the `settings.local.inc.php` file then there's no need to send it through in the Docker container.